src/components/TextField/TextField.tsx

```diff
@@ -0,0 +1,66 @@
+"use client";
+
+import React from "react";
+import clsx from "clsx";
+
+interface TextFieldProps {
+    value: string;
+    onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+    placeholder?: string;
+    size?: "sm" | "md" | "lg";
+    radius?: "none" | "sm" | "md" | "lg";
+    variant?: "outline" | "filled";
+    icon?: "string" | React.ReactNode;
```

Comment:
```suggestion
    icon?: React.ReactNode;
```
기존 코드의 경우, `"string"`이라는 문자열을 허용하는 코드입니다.